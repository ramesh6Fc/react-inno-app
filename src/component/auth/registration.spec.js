import React from 'react';
import '@testing-library/jest-dom/extend-expect';
import renderer from 'react-test-renderer';
import Registration from './registration';


describe('The registration page', () => {
    it ('Registration page renderer as expected',() => {
        const tree = renderer
                        .create(<Registration />)
                        .toJSON();
                        console.log(tree.children, "Registration page elements");

                        expect(tree).toMatchSnapshot ();
    })

})