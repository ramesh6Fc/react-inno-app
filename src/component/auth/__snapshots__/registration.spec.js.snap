// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The registration page Registration page renderer as expected 1`] = `
<div
  className="cards cards-container"
>
  <form
    onSubmit={[Function]}
  >
    <div>
      <div
        className="form-elemt"
      >
        <label
          htmlFor="username"
        >
          Username
        </label>
        <div>
          <input
            className="form-input"
            name="username"
            onBlur={[Function]}
            onChange={[Function]}
            type="text"
            value=""
          />
        </div>
      </div>
      <div
        className="form-elemt"
      >
        <label
          htmlFor="email"
        >
          Email
        </label>
        <div>
          <input
            className="form-input"
            name="email"
            onBlur={[Function]}
            onChange={[Function]}
            type="text"
            value=""
          />
        </div>
      </div>
      <div
        className="form-elemt"
      >
        <label
          htmlFor="password"
        >
          Password
        </label>
        <div>
          <input
            className="form-input"
            name="password"
            onBlur={[Function]}
            onChange={[Function]}
            type="password"
            value=""
          />
        </div>
      </div>
      <div
        className="form-elemt"
      >
        <button
          className="button"
          style={
            Object {
              "cursor": "pointer",
            }
          }
        >
          Sign Up
        </button>
      </div>
    </div>
    
    <button
      disabled={true}
      style={
        Object {
          "display": "none",
        }
      }
    />
  </form>
  <div
    style={
      Object {
        "cursor": "pointer",
        "fontSize": "11px",
        "textAlign": "right",
      }
    }
  >
    <a
      href="!#"
      style={
        Object {
          "color": "#fff",
        }
      }
    >
      Sign In
    </a>
  </div>
</div>
`;
